README.md

```diff
@@ -84,6 +83,36 @@ To run it from your desktop (Windows, Mac, Linux) it can be a bit more involved.
 Once that is done you can open up a command line in the project's directory and run `lime test windows -debug`. Once that command finishes (it takes forever even on a higher end PC), you can run FNF from the .exe file under export\release\windows\bin
 As for Mac, 'lime test mac -debug' should work, if not the internet surely has a guide on how to compile Haxe stuff for Mac.
 
+### FNF now supports [VSCode Devcontainers](https://code.visualstudio.com/docs/remote/containers)
+
+There are some VSCode task created for compiling and running HTML5 versions of FNF inside a devcontainer
+
+### Compiling HTML5 version using Docker
+
+Run this commands:
+
+```bash
+docker build -f .devcontainer/Dockerfile -t fnf:mine .
+docker run --name compile_fnf --rm -v <FULL PATH TO YOUR FNF REPO>:/fnf fnf:mine
+```
+
+If you want to build a HTML5 debug version add `-debug` at the end of docker command

```

Comment:
Grammar and clarity: "To compile a `debug` HTML5 build, append `'-debug` to the end of the Docker command."